src/definitions/modules/dropdown.js

```diff
@@ -3210,12 +3210,12 @@ $.fn.dropdown = function(parameters) {
                 }
                 module.remove.message();
                 if(isUserValue) {
-                  module.remove.value(stringValue);
+                  module.remove.value(stringValue, stringValue, module.get.item(stringValue), preventChangeTrigger);
                   module.remove.label(stringValue);
                 }
                 else {
                   // selected will also remove label
-                  module.remove.selected(stringValue);
+                  module.remove.selected(stringValue, module.get.item(stringValue), preventChangeTrigger);
```

Comment:
Inside the `.remove.selected` function the auto generation of `$selectedItem` is dependent on the `allowAddition` setting. by providing `module.get.item(stringValue)` here directly, this would be ignored and probably won't if `allowAdditions` is true.

If, instead of creating the value by yourself, just provide `false`, this would still work as before (and you dont need to repeat the logic here again) but the additional `preventChangeTrigger` can be provided as awell

```suggestion
                  module.remove.selected(stringValue, false, preventChangeTrigger);
```

Reply:
Thanks. I updated the PR, please review again.